src/Microsoft.Azure.WebJobs.Host/TraceWriter.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.Azure.WebJobs.Host
     /// Base class for trace writers used by the <see cref="JobHost"/> for
     /// event tracing. See <see cref="JobHostConfiguration.Tracing"/> for details.
     /// </summary>
+    [Obsolete("Will be removed in an upcoming version. Use ILogger instead.")]
```

Comment:
Bummer we couldn't remove these for this release.

Reply:
Agreed :(